From 183d077429a22a70a7b4206e23752e5145d9c1a6 Mon Sep 17 00:00:00 2001
From: Heiko Schocher <hs@denx.de>
Date: Mon, 11 May 2020 12:30:03 +0200
Subject: [PATCH 07/19] uboot/build.py: patch sources only if clean

patch the sources in uboot_checkout only if clean is True.

Signed-off-by: Heiko Schocher <hs@denx.de>
---
 tbot/tc/uboot/build.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tbot/tc/uboot/build.py b/tbot/tc/uboot/build.py
index eb7630d..b439cf6 100644
--- a/tbot/tc/uboot/build.py
+++ b/tbot/tc/uboot/build.py
@@ -231,7 +231,8 @@ class UBootBuilder(abc.ABC):
                 path = builder.do_repo_path(host)
 
             repo = builder.do_checkout(path, clean=clean, rev=rev or builder.revision)
-            builder.do_patch(repo)
+            if clean:
+                builder.do_patch(repo)
 
         return repo
 
-- 
2.31.1

